apps/coscrad-frontend/src/components/notes/test-utils/build-dummy-notes.tsx

```diff
@@ -75,7 +75,7 @@ const dualEdgeConnections: INoteViewModel[] = [
     },
     {
         id: '112',
-        note: `This is why term 5 is related to audio transcript 8`,
+        note: `This is why term 5 is related to audio item 8`,
```

Comment:
Maybe we want the sequence numbers here for readability. Otherwise, we should look into why the UUIDs aren't coming through here.